include/CL/cl.h

```diff
@@ -1375,6 +1375,18 @@ clEnqueueTask(cl_command_queue  /* command_queue */,
               const cl_event *  /* event_wait_list */,
               cl_event *        /* event */) CL_EXT_SUFFIX__VERSION_1_2_DEPRECATED;
 
+/* Poclcc OpenCL APIs */
+extern CL_API_ENTRY cl_int CL_API_CALL
+clExportPoclccBinary(cl_program /* program */,
```

Comment:
Ok.
So I will remove those functions. And add what is needed in the device format (which will become the poclcc binary format).